code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -119,15 +119,10 @@
 	if (src.transforming)
 		return
 
-	if (!bioHolder)
```

Comment:
This is set in New() and has no reason to be gone, if sometimes makes it disappear we want to know.